mw-webapp/src/component/editableText/EditableText.tsx

```diff
@@ -116,10 +122,22 @@ export const EditableText = <T extends string | number>(props: EditableTextProps
   };
 
   /**
-   * Update value
+   * Check is value valid or not according min length
    */
-  const updateValue = (updatedValue: string | number) => {
-    setValue(updatedValue as T);
+  const getValidValue = (updateValue: string | number) => {
```

Comment:
If fucntion mane start with get then it shouls always return value, but in this function you don't return any. Need rename

Reply:
I think that old name was ok

Reply:
done